CONTRIBUTING.md

```diff
@@ -16,6 +16,7 @@ contributing to terra, these are documented below.
 * [Changelog generation](#changelog-generation)
 * [Release Notes](#release-notes)
 * [Installing Qiskit Terra from source](#installing-qiskit-terra-from-source)
+* [Building Documentation Locally](#building-documentation-locally)
```

Comment:
Most of the titles don't capitalize all words (the ones that do, refer to concepts such as "Release Notes" or names such as "Lint"), so I'd change the capitalization to:

```suggestion
* [Building documentation locally](#building-documentation-locally)
```